geopandas/tests/test_plotting.py

```diff
@@ -1833,7 +1833,7 @@ def test_column_values():
     # Build test data
     t1 = Polygon([(0, 0), (1, 0), (1, 1)])
     t2 = Polygon([(1, 0), (2, 0), (2, 1)])
-    polys = GeoSeries([t1, t2], index=list("AB"))
+    polys = GeoSeries([t1, t2], index=[0, 1])
```

Comment:
Please keep this intact. If you need another example with a RangeIndex, create another GeoSeries.

Reply:
Thanks, have created a new GeoSeries and GeoDataframe with a numeric index